sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -1,15 +1,23 @@
 # Release History
 
-## 1.1.0-beta.14 (Unreleased)
+## 1.0.19 (2023-02-14)
 
-### Features Added
+### Other Changes
 
-### Breaking Changes
+#### Dependency Updates
 
-### Bugs Fixed
+- Upgraded `azure-core` from `1.35.0` to version `1.36.0`.
+- Upgraded `azure-communication-common` from `1.2.4` to version `1.2.5`.
+
+## 1.1.0-beta.14 (2023-02-14)
```

Comment:
Is this CHANGELOG entry coming from a patch release we already performed? If we are releasing a beta from this branch we should keep the beta changes at the top, I think that's what most people take a look at first.